plutus-pab/src/Plutus/PAB/Core/ContractInstance/BlockchainEnv.hs

```diff
@@ -82,7 +79,7 @@ startNodeClient config instancesState = do
       env' <- STM.atomically $ emptyBlockchainEnv pabRollbackHistory params
       if useDiskIndex && nodeStartsInAlonzoMode pscNodeMode
       then do
-        utxoIx <- Ix.open (unpack dbFile) (Ix.Depth 10) >>= newIORef
+        utxoIx <- Ix.open "./marconi.sqlite" (Ix.Depth 10) >>= newIORef
```

Comment:
Looks bad, we want to use the config path.

Reply:
The issue here is that we no longer have an SQLite path ensured since now it could be a postres configuration.